.golangci.yml

```diff
@@ -72,4 +72,4 @@ linters-settings:
       - gomnd
 
 run:
-  timeout: 20m
+  timeout: 25m
```

Comment:
i'm assuming this is where the `20m` is defined though the error messages seem to call out the `--timeout` command line arg so maybe better suited there 🤔 ?

Reply:
From [the documentation](https://golangci-lint.run/usage/configuration/):

> The config file has lower priority than command-line options. If the same bool/string/int option is provided on the command-line and in the config file, the option from command-line will be used.

so changing the value in this file seems correct.